packages/cli-template-monorepo-ethers/apps/web-app/src/app/globals.css

```diff
@@ -123,43 +156,112 @@ a {
 
 .button:disabled:hover {
     background-color: var(--blue800);
+    background-image: none;
 }
 
 .button-stepper {
     cursor: pointer;
-    color: var(--blue600);
+    color: var(--blue500);
     font-size: 1.1rem;
     border: none;
     background: none;
+    width: 4rem;
+    margin: 0 1rem;
+    display: flex;
+    justify-content: center;
+}
+
+.button-stepper:hover {
+    text-decoration: underline;
+}
+
+.refresh-wrapper {
+    color: var(--slate400);
+    display: flex;
+    align-items: center;
+}
+
+.refresh-wrapper:hover {
+    text-decoration: underline;
 }
 
-.button-link {
+.refresh-button {
     cursor: pointer;
     color: var(--slate300);
     font-size: 1.1rem;
     border: none;
     background: none;
+    padding-right: 1rem;
+    display: flex;
+    align-items: center;
+}
+
+.refresh-button:hover {
+    text-decoration: underline;
+}
+
+.refresh-span {
+    width: 1.5em;
+    height: 1em;
+    display: inline-block;
+    line-height: 1em;
+    -webkit-flex-shrink: 0;
+    -ms-flex-negative: 0;
+    flex-shrink: 0;
+    color: currentColor;
+    vertical-align: middle;
+}
+
+.refresh-icon {
+    margin-inline-end: 0.5rem
+}
+
+.stepper-icon {
+    display: inline-flex;
+    align-self: center;
+    -webkit-flex-shrink: 0;
+    -ms-flex-negative: 0;
+    flex-shrink: 0;
+}
+
+.left-pad {
+    margin-inline-end: 0.5rem;
+}
+
+.right-pad {
+    margin-inline-start: 0.5rem;
+}
+
+.stepper-icon svg {
+    width: 1em;
+    height: 1em;
+    display: inline-block;
+    line-height: 1em;
+    -webkit-flex-shrink: 0;
+    -ms-flex-negative: 0;
+    flex-shrink: 0;
+    color: currentColor;
+    vertical-align:super
 }
 
 .box {
-    padding: 0.8rem;
-    border-style: solid;
-    border-width: 1px;
-    border-color: var(--slate500);
-    border-radius: 4px;
+    border-bottom: 1px solid white;
+    padding: 0.8rem 0;
+    max-height: 50px;
 }
 
 .box-text {
     text-overflow: ellipsis;
     overflow: hidden;
     white-space: nowrap;
-    margin: 0.5rem;
+    font-size: 1rem;
 }
 
 .header {
     display: flex;
     justify-content: space-between;
-    padding: 1.5rem;
+    padding: 0 1.5rem;
+    height: 3.5rem;
 }
 
 .header-left {
```

Comment:
This can be the `header-left` css class to keep the `Feedback` link in the header:

```css
.header-left {
    font-size: 1.125rem;
    font-weight: 500;
    text-decoration: none !important;
    display: flex;
    align-items: center;
}
```